components/case/caseController.js

```diff
@@ -109,49 +114,71 @@ const addCaseArchive = async (req, res, next) => {
       res.status(201).end();
     } else {
       for (let existingCase of existingCases) {
-        if (
-          existingCase.requisitionId == req.body.requisitionId &&
-          arraysEquals(existingCase.limsIds, req.body.limsIds) &&
-          arraysEquals(
+        // check for conflicting changes
```

Comment:
Would it make sense to permit PATCH requests to this endpoint so the client doesn't have to build such a precise request body every time (and the server needn't validate so many fields)?

Reply:
The requests are coming from Shesmu, which is building and submitting case archive actions each time the olive runs, and like the Vidarr /submit endpoint, this endpoint determines whether it is creating or updating a case archive action. 

There are other endpoints which support updates to specific case archive fields.